--- pdnsd-1.2.5/doc/pdnsd.conf.in.query_roots_in_default_conf.patch	2007-05-01 12:13:57.000000000 +0200
+++ pdnsd-1.2.5/doc/pdnsd.conf.in	2007-05-01 12:14:36.000000000 +0200
@@ -14,9 +14,10 @@
 	min_ttl=15m;       # Retain cached entries at least 15 minutes.
 	max_ttl=1w;	   # One week.
 	timeout=10;        # Global timeout option (10 seconds).
 }
 
+/*
 # The following section is most appropriate if you have a fixed connection to
 # the Internet and an ISP which provides good DNS servers.
 server {
 	label= "myisp";
@@ -30,6 +31,7 @@
 	purge_cache=off;   # Keep stale cache entries in case the ISP's
 			   # DNS servers go offline.
 }
+*/
 
 /*
 # The following section is more appropriate for dial-up connections.
@@ -47,7 +49,6 @@
 }
 */
 
-/*
 # This section is meant for resolving from root servers.
 server {
 	label = "root-servers";
@@ -75,7 +76,7 @@
 	policy = included;
 	preset = off;
 }
-*/
+
 
 source {
 	owner=localhost;
