--- pdnsd-1.1.6/doc/Makefile.am.old	Sun Dec 30 00:21:04 2001
+++ pdnsd-1.1.6/doc/Makefile.am		Sun Dec 30 00:21:12 2001
@@ -7,7 +7,7 @@
 
 install-data-hook:
 	$(mkinstalldirs) $(DESTDIR)/$(sysconfdir)
-	# For both source installs and RPM builds
+# For both source installs and RPM builds
 	if test `whoami` = "root"; then \
 	  $(INSTALL) -o 0 -g 0 -m 644 pdnsd.conf $(DESTDIR)/$(sysconfdir)/pdnsd.conf.sample ; \
 	else \
