diff -urN pdnsd-1.2.1.org/configure.in pdnsd-1.2.1/configure.in
--- pdnsd-1.2.1.org/configure.in	2001-12-30 16:32:51.000000000 +0100
+++ pdnsd-1.2.1/configure.in	2005-01-08 20:28:02.797648207 +0100
@@ -387,13 +387,5 @@
 doc/pdnsd.conf
 src/Makefile
 src/pdnsd-ctl/Makefile
-src/rc/Makefile
-src/rc/RedHat/Makefile
-src/rc/RedHat/pdnsd
-src/rc/SuSE/Makefile
-src/rc/SuSE/pdnsd
-src/rc/Debian/Makefile
-src/rc/Debian/pdnsd
-src/rc/Slackware/rc.pdnsd
 src/test/Makefile
 ])
diff -urN pdnsd-1.2.1.org/src/Makefile.am pdnsd-1.2.1/src/Makefile.am
--- pdnsd-1.2.1.org/src/Makefile.am	2001-05-09 19:51:52.000000000 +0200
+++ pdnsd-1.2.1/src/Makefile.am	2005-01-08 20:28:13.437832302 +0100
@@ -9,5 +9,5 @@
 
 ## Try to do this last
 
-SUBDIRS = pdnsd-ctl rc test
+SUBDIRS = pdnsd-ctl test
 
diff -urN pdnsd-1.2.1.org/src/pdnsd-ctl/Makefile.am pdnsd-1.2.1/src/pdnsd-ctl/Makefile.am
--- pdnsd-1.2.1.org/src/pdnsd-ctl/Makefile.am	2001-02-25 01:10:55.000000000 +0100
+++ pdnsd-1.2.1/src/pdnsd-ctl/Makefile.am	2005-01-08 20:28:40.391699012 +0100
@@ -2,9 +2,7 @@
 
 sbin_PROGRAMS = pdnsd-ctl
 
-pdnsd_ctl_SOURCES = pdnsd-ctl.c
-pdnsd_ctl_LDADD =  rr_types.o
-pdnsd_ctl_DEPENDENCIES = rr_types.o
+pdnsd_ctl_SOURCES = pdnsd-ctl.c $(top_srcdir)/src/rr_types.c
 
 # These are Symlinks we want to have in the package
 #EXTRA_DIST = rr_types.h
diff -urN pdnsd-1.2.1.org/src/test/Makefile.am pdnsd-1.2.1/src/test/Makefile.am
--- pdnsd-1.2.1.org/src/test/Makefile.am	2000-11-15 18:27:12.000000000 +0100
+++ pdnsd-1.2.1/src/test/Makefile.am	2005-01-08 20:33:50.000000000 +0100
@@ -6,27 +6,15 @@
 ## Dirty trick:  I demand that these objects be built; then, with the knowledge
 ## that the object files will end up here, I redefine the link chain. 
 
-TESTADDSRC=
-#TESTADDSRC= netdev.c error.c thread.c helpers.c icmp.c
-TESTDEPS = netdev.o error.o thread.o helpers.o icmp.o
-
-TESTOBJS = netdev.o error.o thread.o helpers.o icmp.o
+TESTADDSRC = $(top_srcdir)/src/netdev.c $(top_srcdir)/src/error.c $(top_srcdir)/src/thread.c $(top_srcdir)/src/helpers.c $(top_srcdir)/src/icmp.c
 
 if_up_SOURCES = if_up.c $(TESTADDSRC)
-if_up_LDADD =  $(TESTOBJS)
-if_up_DEPENDENCIES = $(TESTDEPS)
 
 is_local_addr_SOURCES = is_local_addr.c $(TESTADDSRC)
-is_local_addr_LDADD = $(TESTOBJS)
-is_local_addr_DEPENDENCIES = $(TESTDEPS)
 
 tping_SOURCES = tping.c $(TESTADDSRC)
-tping_LDADD = $(TESTOBJS)
-tping_DEPENDENCIES = $(TESTDEPS)
 
 random_SOURCES = random.c $(TESTADDSRC)
-random_LDADD = $(TESTOBJS)
-random_DEPENDENCIES = $(TESTDEPS)
 
 # These are Symlinks we want to have in the package
 #EXTRA_DIST = conff.h error.h helpers.h icmp.h ipvers.h netdev.h thread.h cacheing
