diff -u -r1.3 thread.h
--- src/thread.h	19 May 2001 14:57:30 -0000	1.3
+++ src/thread.h	23 Jun 2002 11:23:35 -0000
@@ -44,6 +44,7 @@
  * under Linux. Also, sigwait() does not seem to work as indicated in the docs */
 #if TARGET==TARGET_LINUX
 #define THREAD_SIGINIT	do { pthread_sigmask(SIG_UNBLOCK,&sigs_msk,NULL);  \
+                             signal(SIGINT,thread_sig);                    \
                              signal(SIGILL,thread_sig);                    \
 	                     signal(SIGABRT,thread_sig);                   \
 	                     signal(SIGFPE,thread_sig);                    \
@@ -51,6 +52,7 @@
 	                     signal(SIGTSTP,thread_sig);                   \
                              signal(SIGTTOU,thread_sig);                   \
                     	     signal(SIGTTIN,thread_sig);                   \
+                             signal(SIGTERM, thread_sig);                  \
                              signal(SIGPIPE, SIG_IGN);                     \
                         } while (0);
 
